loadtest/control/gencontroller/controller.go

```diff
@@ -167,19 +172,19 @@ func (c *GenController) Run() {
 			return
 		}
 
-		if st.get("channels") == c.config.NumChannels {
+		if st.get("channels") >= c.config.NumChannels {
```

Comment:
Was it possible before to create more than one channel in a single action? Or this is just a defensive measure?

Reply:
We can guarantee to create at least the configured number of resources but it can happen to create more because we don't limit the concurrency of the API requests, we only check the state before and after, without locking for the whole thing.

Reply:
Oh my bad, state is shared globally amongst controllers. Yeah makes sense.